cmd/run.go

```diff
@@ -244,6 +244,7 @@ Flags:
 }
 
 func initRuntime(ctx context.Context, params runCmdParams, args []string, addrSetByUser bool) (*runtime.Runtime, error) {
+	params.rt.ServerMode = params.serverMode
```

Comment:
I feel like solutions to this issue should not be dependent on server mode. For example, from this branch, the issue is still present in `opa run`:

```
$ ./opa run bundle2.tar.gz
> data
{
  "pkgs": {
    "pkg2": {
      "name": "pkg2"
    }
  }
}

$ ./opa run bundle2.tar.gz bundle3.tar.gz
error: load error: 1 error occurred during loading: bundle3.tar.gz: merge error

$ ./opa run -s bundle2.tar.gz bundle3.tar.gz
# starts ok
```

Reply:
Hi @charlieegan3 , I removed the ServerMode flag and its working as what examples you gave. But Test steps are failing here in github checks. can you please help. Tests passed before.

Reply:
Thanks @charlieegan3  for clarification. 
So let me know if following behavior is correct ? Lets say bundle4 is a directory containing the rego, data files.
1. opa run bundle2.tar.gz bundle3.tar.gz -b bundle4  // bundle2.tar.gz, bundle3.tar.gz and bundle4 should be loaded as bundles because I gave the -b flag for bundle4
2. opa run bundle2.tar.gz bundle3.tar.gz bundle4  // only bundle2.tar.gz and bundle3.tar.gz should be loaded as bundles

So I think in LoadPaths method some paths should be loaded as bundles and some as normal. Currently everything is either loaded as bundles (the first if clause) OR non-bundles (code after 1st if clause). We have to change that. Let me know if my understanding is correct.

Another question, Is there a way that I can run all the tests in OPA from cmd line.  "make check" command does not work as its given in contributor guide.

Reply:
Hi @charlieegan3 , I have made the change. Now all tests are passing. Please take a look.